### ISSUE TITLE
Festival Team page

### ISSUE THREAD

**Issue by edrushka** [mozilla/mozillafestival.org#653]
https://mozillafestival.org/team
- [x] Add Marc's bio and update photo (completed below).
Title: Senior Production Coordinator

- [ ] Add Chris' bio and photo
Photo below, bio and title still to come

**Comment by edrushka** [comment]
This page can be updated as soon as the copy is ready.

**Comment by mmmavis** [comment]
@hannahkane up for taking this one on?

**Comment by hannahkane** [comment]
Yep! Assigning to me.

**Comment by edrushka** [comment]
@marcwalsh I need your bio and photo please!

**Comment by marcwalsh** [comment]
Biography - @edrushka feel free to edit if you think anything needs changing :)

`Marc is the Senior Production Coordinator for MozFest 2017, and his sixth time working on the festival. Since last year he’s work on various large scale experimental events as well as Glastonbury and the BBC Music Awards. Drawing on this experience, he brings his logistical and production mind-set to help make sure the cogs keep turning across the weekend. `

Image is attached here:
![marc walsh - profile image](https://user-images.githubusercontent.com/30226684/28615393-c6a6ed4a-71f0-11e7-903a-ca97fe8838e7.jpg)

**Comment by edrushka** [comment]
Made tiny tweaks to Marc's bio. Here is final text:

Marc is the Senior Production Coordinator for MozFest 2017, his sixth year with the festival. Since last year he’s worked on various large scale experimental events as well as Glastonbury and the BBC Music Awards. Drawing on this experience, he brings his logistical and production mind-set to help make sure the cogs keep turning across the weekend.

**Comment by edrushka** [comment]
Unblocking. Chris' addition will have to come later on a separate ticket.

**Comment by edrushka** [comment]
![chris](https://user-images.githubusercontent.com/4515662/28846232-83497f8a-76bf-11e7-89d3-efbf55468bed.jpeg)

Chris Lawrence
As Vice President, Mozilla Leadership Network, Chris oversees the development and expansion of a global network of leaders for Internet health. These efforts have reached hundreds of thousands of people in over 90 cities across the globe. Chris has never missed a Mozilla Festival!

[^ don't provide a festival job title]

**Comment by edrushka** [comment]
Oops - @hannahkane just saw that you already merged Marc's bio. Do you want me to file a new ticket for Chris?

**Comment by hannahkane** [comment]
No, that's okay @edrushka. I'll file a PR for this one now, and once it's merged, it will close this ticket.

**Comment by hannahkane** [comment]
No job title? I'm just going to put Chief Punk Rock Enthusiast. Or Resident Eyebrow Raiser.

JK.
